src/view/components/cpx/CpxImage.tsx

```diff
@@ -20,30 +20,34 @@ interface IProps {
     onMouseLeave: (button: HTMLElement, event: Event) => void;
 }
 
-let firstTime = true;
-
-// Functional Component render
-const CpxImage: React.FC<IProps> = props => {
-    const svgElement = window.document.getElementById("cpx_svg");
-
-    if (svgElement) {
-        if (firstTime) {
-            initSvgStyle(svgElement, props.brightness);
-            setupButtons(props);
-            setupPins(props);
-            setupKeyPresses(props.onKeyEvent);
-            setupSwitch(props);
-            firstTime = false;
+//
+export class CpxImage extends React.Component<IProps, any> {
+    componentDidMount() {
+        const svgElement = window.document.getElementById("cpx_svg");
+        if (svgElement) {
+            initSvgStyle(svgElement, this.props.brightness);
+            setupButtons(this.props);
+            setupPins(this.props);
+            setupKeyPresses(this.props.onKeyEvent);
+            setupSwitch(this.props);
+            this.updateImage();
+        } else {
+            console.log("Cannot find svg");
```

Comment:
Should we do more than just console.log()? Should we even include console.log(). Not very familiar with this situation to be honest

Reply:
Removed the console log for now